src/azul/service/manifest_service.py

```diff
@@ -1394,7 +1394,11 @@ def command_lines(cls,
         file_options = [
             '--fail-early',  # Exit curl with error on the first failure encountered
             '--continue-at -',  # Resume partially downloaded files
-            '--retry 2',  # Retry a file download up to X times on transient error
+            # Since the wAF rate rule is evaluated every 30 sec, and it looks at
+            # the request count over the last 5 min, we specify 7.5 min worth of
```

Comment:
```suggestion
            # the request count over the last 5 min, we specify 7.5 min worth of
```

5 min is also a parameter